infrastructure/code/template/details.html

```diff
@@ -60,6 +60,10 @@
       overflow-y: auto;
     }
 
+    section {
+      padding: 10px;
+    }
+
```

Comment:
Looking great! 
Do you think we could add some rules to the Data Flow section to make it horizontally scrollable? 

I quickly tested these [rules](https://www.geeksforgeeks.org/make-a-div-horizontally-scrollable-using-css/) in VSCode and IntelliJ and I compared with and without the rules and they look like the following:


https://github.com/snyk/snyk-ls/assets/1948377/de8492f3-2a85-4548-aec8-488143b4837b


https://github.com/snyk/snyk-ls/assets/1948377/e83f7b4c-68c4-4da9-b100-16a7c4553898



For IntelliJ we would need to style the scrollbar too 😬

Reply:
I remember we have a ticket to take care of this: https://snyksec.atlassian.net/browse/IDE-256. It would make sense to make it consistent with what's already in there in the old UI, which I would need to check if it supports scrolling. I think we could also just wrap the text to the next line if it's too long, which is what I preferred. But basically I think I'd prefer to do it in another ticket if that's okay?

Reply:
It's approved! As you like 🙌 
Thank you